ui/page/root/trade_page.go

```diff
@@ -19,24 +17,28 @@ const (
 	TradePageID = "Trade"
 )
 
+var tabTitles = []string{
+	values.String(values.StrCentralizedExchange),
+	values.String(values.StrDcrDex),
```

Comment:
I think DCRDEX should be the first here because we proly want users to see that first. What do you think @dreacot?

Reply:
Alright, that's fine